e2e/rdctl.e2e.spec.ts

```diff
@@ -1375,18 +1375,41 @@ test.describe('Command server', () => {
     test('should verify nerdctl can talk to containerd', async() => {
       const { stdout } = await rdctl(['list-settings']);
       const settings: Settings = JSON.parse(stdout);
+      const navPage = new NavPage(page);
 
       if (settings.containerEngine.name !== ContainerEngine.CONTAINERD) {
         const payloadObject: RecursivePartial<Settings> = {
           version:         CURRENT_SETTINGS_VERSION,
           containerEngine: { name: ContainerEngine.CONTAINERD },
         };
-        const navPage = new NavPage(page);
 
         await tool('rdctl', 'api', '/v1/settings', '--method', 'PUT', '--body', JSON.stringify(payloadObject));
         await expect(navPage.progressBar).not.toBeHidden();
         await navPage.progressBecomesReady();
         await expect(navPage.progressBar).toBeHidden();
+        try {
+          await tool('nerdctl', 'info');
+        } catch (ex:any) {
+          const { stdout, stderr, message } = ex;
+          console.log(`<nerdctl info> failed: stdout: ${ stdout }, stderr: ${ stderr }, message: ${ message }`);
```

Comment:
This message should be changed to indicate that it's expected (since we're running not-containerd).

Reply:
But at line 1412 we make an API call to change the containerEngine to containerd, and then we wait for the progress bar to start running, and then stop. So t line 1417 we should be running containerd.

Reply:
You're swallowing the exception (and making it not fail the test), hence I'm calling it "expected" even though this isn't the desired code path.  We should either ① cause the test to fail (hence including "failed:" is okay), or ② not fail the test, and _not_ say "failed:".

Reply:
Good point. I changed the code to be more explicit when we need to check for the socket, and force a restart if the socket isn't readable.

And again, don't merge this code because it papers over an existing issue.